payments/models.py

```diff
@@ -36,6 +38,60 @@ def __setattr__(self, key, value):
         self._payment.extra_data = json.dumps(data)
 
 
+class BaseSubscription(models.Model):
+    token = models.CharField(
+        _("subscription token/id"),
+        help_text=_("Token/id used to identify subscription by provider"),
+        max_length=255,
+        default=None,
+        null=True,
+        blank=True,
+    )
```

Comment:
I think, that some kind of token or payment identification would be needed for most implementation.
I used this is for Subscription ID in case of PayPal and for Card token in case of PayU. Accessing that through JSONField would be quite unflexible.

But adding JSONField for additional fields might be a good idea.

Reply:
> But adding JSONField for additional fields might be a good idea. 

Do you want to add those on this PR, or follow-up later?